lib/utils.pm

```diff
@@ -1830,7 +1830,8 @@ This functions checks if ca-certificates-suse is installed and if it is not it a
 sub ensure_ca_certificates_suse_installed {
     return unless is_sle;
     if (script_run('rpm -qi ca-certificates-suse') == 1) {
-        my $distversion = get_required_var("VERSION") =~ s/-SP/_SP/r;    # 15 -> 15, 15-SP1 -> 15_SP1
+        my $host_version = get_var("HOST_VERSION") ? get_var("HOST_VERSION") : get_var("VERSION");
```

Comment:
```suggestion
        my $host_version = get_var("HOST_VERSION", get_var("VERSION"));
```

Reply:
:) this is what i did and i submitted wrong. awesome. let me fix it. One question tho. Do you have any other places that this variable can be used inside the container spectrum which can be adjust respectively? We call other libraries also so i am afraid that it might be hidden somewhere else as well

Reply:
> :) this is what i did and i submitted wrong. awesome. let me fix it. One question tho. Do you have any other places that this variable can be used inside the container spectrum which can be adjust respectively? We call other libraries also so i am afraid that it might be hidden somewhere else as well

Wait. actually this is what i have done in the beginning but i changed it. i think this is more clear. and i just notice a small thingy. To address it i need to do something different. do you mind to close this and comment to the next commit?